### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by thebiblelover7** [genonbeta/TrebleShot-Desktop#38]
Is this project still active? The last commit was in 2019. I hope that there is some work in progress behind the scenes.

**Comment by velitasali** [comment]
Yes, it indeed is. It is just that most of the work has gone into the Android version and libraries that the project use. I will continue working on this when I am able to release a newer Android version that contains those changes.
